README.md

```diff
@@ -133,10 +133,14 @@ Each piece of code (change/patch) is verified, to make the change is safe to mer
 
 <details>
 <summary>What is Continuous Deployment?</summary><br><b>
+
+A development strategy used by developers to release software automatically into production where any code commit must pass through an automated testing phase. Only when this is successful is the release considered production worthy. This eliminates any human interaction and should be implemented only after prodcution-ready pipelines have been set with real-time monitoring and reporting of deployed assets. If any issues are detected in production it should be easy to rollback to previous working state. 
```

Comment:
Did you wrote it by yourself or copied it from somewhere? If copied, please mention in the answer where the answer is taken from (with a link). Thanks.

Reply:
Not answered entirely by myself but will insert link with the long explanation.